config/plugin.config.ts

```diff
@@ -19,7 +21,7 @@ function getModulePackageName(module: { context: string }) {
   return packageName;
 }
 
-export default (config: any) => {
+export default (config: IWebpackChainConfig) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4604636)